lib/WorkflowMain.groovy

```diff
@@ -33,6 +33,8 @@ class WorkflowMain {
 
         // Check that a -profile or Nextflow config has been provided to run the pipeline
         NfcoreTemplate.checkConfigProvided(workflow, log)
+        // Check that the profile doesn't contain spaces and doesn't end with a trailing comma
+        checkProfile(workflow.profile, args, log)
```

Comment:
looks like you are missing half of what was added in this [PR](https://github.com/nf-core/tools/pull/2678/files). NOt sure why though

Reply:
added

Reply:
I have a theory.....